lhotse/recipes/ami.py

```diff
@@ -339,8 +339,8 @@ def _parse_href(href, wid_to_word):
                     continue
                 start_time = float(seg.attrib["transcriber_start"])
                 end_time = float(seg.attrib["transcriber_end"])
-                assert len(seg.getchildren()) == 1, "Multiple child segments found"
-                seg_child = seg.getchildren()[0]
+                assert len(list(seg)) == 1, "Multiple child segments found"
```

Comment:
```suggestion
                assert len(seg) == 1, "Multiple child segments found"
```

Please see the doc at https://docs.python.org/3/library/xml.etree.elementtree.html

> Element objects also support the following sequence type methods for working with subelements: `__delitem__()`, `__getitem__()`, `__setitem__()`, `__len__().`